data/en/test_average_less_validation_against_total.json

```diff
@@ -0,0 +1,93 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "survey_id": "0",
+    "title": "Grouped validation against total test survey",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "A survey that tests the average of breakdown question answers against a total",
+    "sections": [{
+        "id": "default-section",
+        "groups": [{
+            "id": "group",
+            "title": "Validate average against total test",
+            "blocks": [{
+                    "type": "Question",
+                    "title": "Target Total",
+                    "id": "total-block",
+                    "description": "",
+                    "questions": [{
+                        "id": "total-question",
+                        "title": "Total",
+                        "type": "General",
+                        "answers": [{
+                            "id": "total-answer",
+                            "label": "Total",
+                            "mandatory": true,
+                            "type": "Number",
+                            "decimal_places": 2,
+                            "max_value": {
+                                "value": 100
+                            }
+                        }]
+                    }]
+                },
+                {
+                    "type": "Question",
+                    "title": "Calculated Total",
+                    "id": "breakdown-block",
+                    "questions": [{
+                        "id": "breakdown-question",
+                        "title": "Breakdown",
+                        "type": "Calculated",
+                        "calculated": {
+                            "calculated_type": "average",
+                            "answer_id": "total-answer",
+                            "answers_to_group": [
+                                "breakdown-1",
+                                "breakdown-2",
+                                "breakdown-3",
+                                "breakdown-4"
+                            ],
+                            "condition": "less-than"
+                        },
+                        "answers": [{
+                                "id": "breakdown-1",
+                                "label": "Breakdown 1",
+                                "mandatory": false,
+                                "decimal_places": 2,
+                                "type": "Number"
```

Comment:
Consider using answer type="Currency" for at least one of these test schemas. Currency fields are the primary use case.